chall01/vtran.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+import sys
+
+tittut = {
+    'a': ".-",
+    'b': "-...",
+    'c': "-.-.",
+    'd': "-..",
+    'e': ".",
+    'f': "..-.",
+    'g': "--.",
+    'h': "....",
+    'i': "..",
+    'j': ".---",
+    'k': "-.-",
+    'l': ".-..",
+    'm': "--",
+    'n': "-.",
+    'o': "---",
+    'p': ".--.",
+    'q': "--.-",
+    'r': ".-.",
+    's': "...",
+    't': "-",
+    'u': "..-",
+    'v': "...-",
+    'w': ".--",
+    'x': "-..-",
+    'y': "-.--",
+    'z': "--..",
+    ' ': " "
+}
```

Comment:
Would have been better to write it in one-line and use a meaningful variable name